examples/ConstraintIB/eel3d/eelgenerator3d.cpp

```diff
@@ -226,7 +226,7 @@ main()
     }
     std::fstream eelstream;
     eelstream.open("eel3d.vertex", std::fstream::out);
-    assert((unsigned)total_lag_pts == shape_new[0].size());
+    assert(static_cast<unsigned>(total_lag_pts) == shape_new[0].size());
```

Comment:
This is still a bit weird (but not incorrect) since on the right we have `std::size_t`. Could you use that instead of `unsigned` here?